### DISCUSSION TITLE
Warn if there are pending review comments when merging a PR

### DISCUSSION THREAD

**Discussion by rmjarvis** [community/community#61409] [Pull Requests] [answered]
### Select Topic Area

Product Feedback

### Body

If someone is in the middle of a code review and has accumulated a bunch of comments, there is no way it seems for anyone else to know that.  Normally, this is perfectly fine.  They get notified when all the comments are eventually submitted.  However, if someone thinks that all reviews are done, they may think it's ok to merge the PR before the pending comments are submitted.  This is annoying for all involved.

It would be really nice if when clicking merge, the person could be notified with a message like "Mike Jarvis has pending review comments.  Are you sure you want to merge before they have submitted their review?"  This would let the person know who to contact about whether and when they plan to submit the review, and they can still decide to go ahead an merge anyway if that is deemed appropriate.

BTW, a similar request was made on [this very old issue site](https://github.com/isaacs/github/issues/996), so this is not a new idea.  But one that just bit me today, so I'd like to see some action on it.

**Comment by MadelineRitchie** [comment]
See also https://github.com/orgs/community/discussions/34607

**Comment by rmjarvis** [comment, marked as answer]
Also https://github.com/orgs/community/discussions/7552
Apparently, I should have been more thorough in my searching before opening a new discussion.
